### PR TITLE
Started gd translation.

### PR DESCRIPTION
**Pull request by gunchleoc**
Added new locale and started translation. I haven't gotten very far yet, because I can't select any language for testing.

### CODE REVIEW

**Comment by Bertram25** [comment]
Hey @gunchleoc ,

Thanks for translation effort! :)

As for your translation issue, if and only if you are on linux, it is true that sometimes the translation don't get updated properly the first time.
After modifying the LINGUAS file, you'll need to get rid of your CMakeCache.txt file and the CMakeFiles/ folder and reinvoke CMake again. Then, during the 'make' step, you should see a line looking like: `Generating gd.gmo`

If you are on windows, you'll have to install poedit, open the gd.po file with it and save. This will create a .mo file next to the .po file.
You'll then have to copy this file as "valyriatear.mo" into the folder `<repository_root_folder>\po\gd\LC_MESSAGES`
E.g.: `C:\Users\Me\dev\ValyriaTear\po\gd\LC_MESSAGES\valyriatear.mo`
Then, you should be able to make the language work.

I hope this will help you. Cheers,

**Comment by gunchleoc** [comment]
Since my language only has 2 localizers for anything at all, I'll wait for the engine bug to be fixed before carrying on. Sorry, but I do have to prioritize translations that are available cross-platform. I really like this game though, so if you would add my language, it will be easier for me to carry on in the future.

**Comment by Bertram25** [comment]
Well, as translation are perfectly working on my end, let's work on making me understand the bug on your side.
But, to make it clear, Valyria Tear's build system is absolutely not the same than Hero of Allacrost, AFAIK. So, the bug they have and mine may totally be different.

**Comment by Bertram25** [comment]
Let me assess a few things:

- Are on Windows?
- Did you clone and compiled ValyriaTear yourself, or is it from an instance used from the Windows installer?

**Comment by gunchleoc** [comment]
What I did:

- Download VT from homepage
- Install on Windows 7, 64 bit
- Select any language other than English, for example German

Result: All UI strings are in English.

**Comment by gunchleoc** [comment]
Is there anything blocking this pull request? I would like to try fixing some stuff in the source code (I am having serious string truncation issues that can't be solved by translation changes), and I made the mistake of not putting this into a branch.

I also found the Transifex page and will continue translating there once my language is added.

File: data/config/languages.lua [hunk 1]
```diff
@@ -36,6 +36,10 @@ languages = {
         name = "Galego",
         interwords_spaces = true
     },
+    ["gd"] = {
```
Comment (akien-mga) [inline data/config/languages.lua]: It should be between `fr` and `gl` alphabetically speaking, though I'm not sure it matters.

**Comment by gunchleoc** [comment]
Good point. Nederlands was also in the wrong order, so I've fixed them both.

File: ValyriaTear.nsi [hunk 2]
```diff
@@ -72,6 +72,8 @@ Section "MainSection" SEC01
   File /oname=valyriatear.mo po\es.mo
   SetOutPath "$INSTDIR\po\gl\LC_MESSAGES"
   File /oname=valyriatear.mo po\gl.mo
+  SetOutPath "$INSTDIR\po\gd\LC_MESSAGES"
```
Comment (akien-mga) [inline ValyriaTear.nsi]: Also wrong alphabetical order (but that's the case for `en_GB` too, so likely not an issue.

File: po/LINGUAS [hunk 3]
```diff
@@ -4,6 +4,7 @@ de
 es
 fr
 gl
+gd
```
Comment (akien-mga) [inline po/LINGUAS]: Here too ;)

**Comment by Bertram25** [comment]
Hey @gunchleoc ,

for the record, it is working on Linux with the given PR: :D
image

**Comment by Bertram25** [comment]
@akien-mga So nice to see you around! :)

**Comment by Bertram25** [comment]
@gunchleoc  It's nice you made it work btw. Do you want me to consider the merge of your PR right now, and do you prefer filling the po file more before doing so?

**Comment by akien-mga** [comment]
@Bertram25 I'd say merge first, complete later: https://github.com/ValyriaTear/ValyriaTear/pull/546#issuecomment-287287293

**Comment by gunchleoc** [comment]
Fixed the language order in the other 2 files. From my end, this is ready to go now.

My translation is about half-way through now, but I think it makes more sense to continue translation on Transifex.

**Comment by Bertram25** [comment]
Ah sure. Missed that one, sorry.
Nice to see some participation coming in. :)

**Comment by Bertram25** [comment]
> My translation is about half-way through now, but I think it makes more sense to continue translation on Transifex.

If not done already, I'll upload the translation on transifex to let you keep on there.

**Comment by gunchleoc** [comment]
No need to upload, just add me and I will upload myself :)

**Comment by Bertram25** [comment]
oops, I uploaded the gd translation files from the po you had in the PR and sent you an invite to manage the gd language: https://www.transifex.com/valyria-tear/valyria-tear/language/gd/

Have fun! :)

**Comment by gunchleoc** [comment]
For some reason I'm not in yet, so I have sent a team join request.

**Comment by Bertram25** [comment]
Should be ok now. Feel free to tell me if something's wrong.

**Comment by gunchleoc** [comment]
I'm in now, thanks!

**Comment by Bertram25** [comment]
perfect!

**Pull request merged**